test/integration/client-side-encryption/driver.test.ts

```diff
@@ -471,3 +476,89 @@ describe('Range Explicit Encryption with JS native types', function () {
     });
   });
 });
+
+describe('CSOT', function () {
+  describe('Explicit Encryption', function () {
+    describe('#createEncryptedCollection', function () {
+      let keyVaultClient: MongoClient;
+      let internalClient: MongoClient;
+      let clientEncryption: ClientEncryption;
+      const LOCAL_KEY = Buffer.from(
+        'Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk',
+        'base64'
+      );
+
+      beforeEach(async function () {
+        internalClient = this.configuration.newClient();
+        await internalClient.connect();
+        await internalClient.db('keyvault').createCollection('datakeys');
+        keyVaultClient = this.configuration.newClient({ timeoutMS: 100 });
+        clientEncryption = new ClientEncryption(keyVaultClient, {
+          keyVaultNamespace: 'keyvault.datakeys',
+          kmsProviders: { local: { key: LOCAL_KEY } }
+        });
+        await internalClient
+          .db()
+          .admin()
+          .command({
+            configureFailPoint: 'failCommand',
+            mode: {
+              times: 1
+            },
+            data: {
+              failCommands: ['create'],
+              blockConnection: true,
+              blockTimeMS: 1000
+            }
+          } as FailPoint);
+      });
+
+      afterEach(async function () {
+        await internalClient
+          .db()
+          .admin()
+          .command({
+            configureFailPoint: 'failCommand',
+            mode: 'off'
+          } as FailPoint);
+        await internalClient
+          .db('db')
+          .collection('newnew')
+          .drop()
+          .catch(() => null);
+        await internalClient.close();
+        await keyVaultClient.close();
+      });
+
+      it(
+        'times out due to timeoutMS',
+        {
+          requires: {
+            clientSideEncryption: true,
+            mongodb: '>=7.0.0',
+            topology: '!single'
+          }
+        },
+        async function () {
+          const createCollectionOptions = {
+            encryptedFields: { fields: [{ path: 'ssn', bsonType: 'string', keyId: null }] }
+          };
+
+          const db = internalClient.db('db');
+          const err = await clientEncryption
+            .createEncryptedCollection(db, 'newnew', {
+              provider: 'local',
+              createCollectionOptions,
+              masterKey: null
+            })
+            .catch(err => err);
+
+          expect(err).to.be.instanceOf(MongoCryptCreateEncryptedCollectionError);
+          expect(err.message).to.contain(
```

Comment:
We usually test for `instanceof` - here I think you'd want to check for `(err.cause).to.be.instanceOf(...)`.